docs/en/modules/wifi.md

```diff
@@ -398,6 +398,9 @@ none
 
 Registers callbacks for WiFi station status events.
 
+!!! note
+    Please update your program to use the [`wifi.eventmon`](#wifieventmon-module) API, as the `wifi.sta.eventmon___()` API is to be depreciated. 
```

Comment:
> is to be depreciated

1.  That should be "deprecated" I guess?
2.  "to be" confuses me a little here, is it deprecated or not?